component/src/main/java/org/wso2/extension/siddhi/store/influxdb/InfluxDBStore.java

```diff
@@ -0,0 +1,565 @@
+                                "insert into StockTable ;",
+                        description = " The above example creates a table in the  database if it does not exist" +
+                                " already with 'symbol which is configured under '@Index' annotation  as a tag key " +
+                                "and 'volume' and 'price' as field keys . Time represents the timestamp" +
+                                "  associated with a particular data."
+
+                ),
+                @Example(
+                        syntax = " @Store(type = \"influxdb\",\n" +
+                                "   url = \"http://localhost:8086\",\n" +
+                                "   username = \"root\",\n" +
+                                "   password = \"root\" ,\n" +
+                                "   database =\"aTimeSeries\")\n" +
+                                "@Index(\"symbol\",\"time\")\n" +
+                                "define table StockTable(symbol string,volume long,price float,time long) ;\n" +
+                                "@info(name = 'query4')\n" +
+                                "from ReadStream#window.length(1) join StockTable on " +
+                                "StockTable.symbol==ReadStream.symbols \n" +
+                                "select StockTable.symbol as checkName, StockTable.price as checkCategory,\n" +
+                                "StockTable.volume as checkVolume,StockTable.time as checkTime\n" +
+                                "insert into OutputStream; ",
+
+                        description = " The above example creates a table in the database if it does not exist" +
+                                " already with 'symbol' which is configured under '@Index' annotation  as a " +
+                                "tag key , 'volume' and 'price' as field keys and time represents the timestamp" +
+                                "Then the table is joined with a stream named 'ReadStream' based on a condition."
+                )
+        }
+)
+
+public class InfluxDBStore extends AbstractQueryableRecordTable {
+
+    private static final Log log = LogFactory.getLog(InfluxDBStore.class);
+
+    private String tableName;
+    private String database;
+    private String url;
+    private String username;
+    private String password;
+    private int timePosition;
+    private Annotation storeAnnotation;
+    private Annotation indices;
+    private InfluxDB influxdb;
+    private List<String> attributeNames;
+    private List<Integer> tagPositions;
+
+    /**
+     * Initializing the Record Table
+     *
+     * @param tableDefinition definintion of the table with annotations if any
+     * @param configReader    this hold the {@link AbstractQueryableRecordTable} configuration reader.
+     */
+
+    @Override
+    protected void init(TableDefinition tableDefinition, ConfigReader configReader) {
+
+        attributeNames =
```

Comment:
use this.attributeName as best practice when accessing class level variable,
```suggestion
        this.attributeNames =
```

Reply:
https://github.com/wso2-extensions/siddhi-store-influxdb/pull/1/commits/520b093086a08e2d8d9df4297dcd71914e15813b

Reply:
@malkisandamini Just a small concern. This has being addressed for `attributeNames` only while there are other class variables as well. If you accept this practice, then let's be consistent and do it for all the class level variables :)